external-crates/move/move-compiler/tests/move_check/typing/loop_result_type.exp

```diff
@@ -14,3 +14,11 @@ warning[W09005]: dead or unreachable code
    │
    = This warning can be suppressed with '#[allow(dead_code)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
 
+warning[W09005]: dead or unreachable code
+   ┌─ tests/move_check/typing/loop_result_type.move:26:9
+   │
+26 │         x
+   │         ^ Unreachable code. This statement (and any following statements) will not be executed.
```

Comment:
It feels a bit strange to give this where when we already have the `Invalid use of a divergent expression` error above. That error is just a flavor of a dead code error. Once we give one, we shouldn't give anymore